Library/Formula/postgresql.rb

```diff
@@ -1,9 +1,11 @@
 class Postgresql < Formula
   homepage "http://www.postgresql.org/"
+  revision 1
 
   stable do
     url "http://ftp.postgresql.org/pub/source/v9.4.0/postgresql-9.4.0.tar.bz2"
     sha256 "7a35c3cb77532f7b15702e474d7ef02f0f419527ee80a4ca6036fffb551625a5"
+    version "9.4"
```

Comment:
Blurgh. Yeah. If I remove that particular tiny chunk, the problem goes away. Sigh. Maybe we can just handle that one tiny tiny line manually :/.